### ISSUE TITLE
Fail to sync transactions of a miner

### ISSUE THREAD

**Issue by Keith-CY** [nervosnetwork/neuron#2941]
This issue comes from the following cases
1. https://github.com/nervosnetwork/neuron/issues/2940
2. https://github.com/nervosnetwork/neuron/issues/2939
3. https://talk.nervos.org/t/neuron-stuck-syncing-at-block-5-188-000/7526/4

---
The following error message were found
```log
1. [2023-11-15T13:10:49.139Z] [warn]  function call error: Error: transaction 0xde1b1b3b2deda3a7759912e86e3808464edbe42b2ffac27204cdad5cbf7d8491 status: added, retry 2 ...
```
```log
2. [2023-11-15T08:13:52.127Z] [warn]  function call error: Error: transaction 0xb3719258ab9d595935d50b61b5ff9a5ceb07a41bf44d4562559bd76d74d6f48b status: added, retry 3 ...
```
```log
3. [2023-11-15T09:01:13.441Z] [error] retry saving transactions in 2 seconds due to error: Error: failed to fetch transaction for hash 0x351b9ba************fd96b1d1b
```

Noted that case 3 shows `failed to fetch transaction for hash ...` in the error message but others don't

PS: `retry saving transactions in 2 seconds due to error:` is case 3 is added by Neuron, and the following is proxied from the light client

---

After inspection, I found all these transactions are related to the address `ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqwau7qpcpealv6xf3a37pdcq6ajhwuyaxgs5g955`(https://explorer.nervos.org/address/ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqwau7qpcpealv6xf3a37pdcq6ajhwuyaxgs5g955)

One of them is a transaction to distribute rewards to miners, and two of them are cellbase transactions.

**Comment by Keith-CY** [comment]
Hi @gpBlockchain
Could you help verify the synchronization of `ckb1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqwau7qpcpealv6xf3a37pdcq6ajhwuyaxgs5g955` with light client? We will focus on checking logic inside Neuron for this case.

**Comment by Keith-CY** [comment]
The reason has been found at https://github.com/Magickbase/neuron-public-issues/issues/322

**Comment by itsfranzsis** [comment]
When will this issue be fixed ?

**Comment by Keith-CY** [comment]
> When will this issue be fixed ?

Hi @itsfranzsis sorry for replying late, it has been fixed in Neuron@v0.112.0 at https://github.com/nervosnetwork/neuron/releases/tag/v0.112.0
